src/rules/at-rule-name-space-after/__tests__/index.js

```diff
@@ -0,0 +1,308 @@
+    code: "@my-ruleset: { .my-selector { background-color: black; } };",
+  }, {
+    code: ".class1 { .mixin(#ddd) }",
+    description: "ignore mixins",
+  }, {
+    code: ".button { &-ok { } }",
+    description: "ignore parent selectors",
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  syntax: "scss",
+  config: ["always"],
+
+  accept: [{
+    code: "@mixin mixin() { @content; }; .colors { @include mixin { color: $color; }}",
+    description: "ignore content blocks",
+  }],
+})
+
+testRule(rule, {
+  ruleName,
+  config: ["always-single-line"],
+
+  accept: [ {
+    code: "@charset \"UTF-8\";",
+  }, {
+    code: "@charset\n\"UTF-8\";",
+  }, {
+    code: "@charset\r\n\"UTF-8\";",
+  }, {
+    code: "@charset\n\n\"UTF-8\";",
+  }, {
+    code: "@charset\r\n\r\n\"UTF-8\";",
+  }, {
+    code: "@charset \"UTF-8\";",
+  }, {
+    code: "@charset\n\"UTF-8\";",
+  }, {
+    code: "@charset\r\n\"UTF-8\";",
+  }, {
+    code: "@charset\n\n\"UTF-8\";",
+  }, {
+    code: "@charset\r\n\r\n\"UTF-8\";",
+  }, {
+    code: "@import \"x.css\";",
+  }, {
+    code: "@import \"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import url(\"x.css\");",
+  }, {
+    code: "@import\nurl(\"x.css\");",
+  }, {
+    code: "@import\r\nurl(\"x.css\");",
+  }, {
+    code: "@import url(\"x.css\") screen and (orientation:landscape);",
+  }, {
+    code: "@namespace url(XML-namespace-URL);",
+  }, {
+    code: "@media (min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@media (min-width: 700px) and (orientation: landscape)  { }",
+  }, {
+    code: "@media (min-width: 700px) and (orientation: landscape)\n{ }",
+  }, {
+    code: "@media (min-width: 700px) and (orientation: landscape)\r\n{ }",
+  }, {
+    code: "@media (min-width: 700px)  and (orientation: landscape) { }",
+  }, {
+    code: "@media (min-width: 700px)\nand (orientation: landscape) { }",
+  }, {
+    code: "@media (min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@media(min-width: 700px)\nand (orientation: landscape) { }",
+  }, {
+    code: "@media(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@media(min-width: 700px) and\n(orientation: landscape) { }",
+  }, {
+    code: "@media(min-width: 700px) and\r\n(orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@supports (animation-name: test) { }",
+  }, {
+    code: "@keyframes identifier { }",
+  }, {
+    code: "@viewport { }",
+  }, {
+    code: "@viewport{ }",
+  }, {
+    code: "@viewport\n{ }",
+  }, {
+    code: "@viewport\r\n{ }",
+  }, {
+    code: "@viewport\n\n{ }",
+  }, {
+    code: "@viewport\r\n\r\n{ }",
+  }, {
+    code: "@counter-style winners-list { }",
+  }, {
+    code: "@font-face { };",
+  }, {
+    code: "@unknown \"ident\";",
+  }, {
+    code: "@unknown ident { };",
```

Comment:
I think it's worth allowing non-standard at-rules that for whatever reason don't have any parameters, e.g. `@content;` or `@foobar;`. Should be easy to ignore those and prevent future complaints.